Makefile

```diff
@@ -69,6 +69,11 @@ npm-init:
 npm-build:
 	npm run build;
 
+# FAKING THIS SO IT DOESN"T FEEL LIKE MULTIPLE RUNS OF THE SAME COMMAND
+.PHONY: init
+.SILENT: init
+init: go-mod config-aws config-mdai config-otel
```

Comment:
```suggestion
init: go-mod config-aws config-mdai
```

Reply:
What do we need this target for? No needs to run same command multiple times. With the current flow it's just 

$ make cert
$ make config
$ make install

Am I missing something here?